shared/src/commonMain/kotlin/com/mongodb/dublinmug_kmm/RealmRepo.kt

```diff
@@ -30,37 +34,34 @@ class RealmRepo {
         val config = SyncConfiguration
             .Builder(user, setOf(QueryInfo::class))
             .initialSubscriptions { realm ->
-                // only can write data, which cover in initialSubscriptions
-                add(
-                    query = realm.query<QueryInfo>(),
-                    name = "subscription name",
-                    updateExisting = true
-                )
+                // Only data covered by a subscription can be written to the realm.
+                add(query = realm.query<QueryInfo>())
```

Comment:
You can also replace this with `realm.query<QueryInfo>().subscribe()`, but I opted for the `add` method as that makes it more visible what happens.